tools/run_tests/run_tests.py

```diff
@@ -57,6 +57,9 @@
 _ROOT = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), '../..'))
 os.chdir(_ROOT)
 
+# Avoid running too many tests in parrallel (e.g., to not overload port server).
+# TODO: possibly tune this.
+_MAX_PARALLEL_TESTS_CPU_AGNOSTIC = 32
```

Comment:
once we start using 32core and 64core workers, the default of 32 seems insufficient. 
Would things still work fine with _MAX_PARALLEL_TESTS_CPU_AGNOSTIC = 64?